src/main/scala/chiseltest/simulator/ChiselBridge.scala

```diff
@@ -154,7 +154,8 @@ private object ChiselBridge {
     // ignoreDecodeTableAnnotation since it is not needed by the firrtl compiler
     case _: DecodeTableAnnotation => None
     //
-    case _ => throw new NotImplementedError(s"TODO: convert ${anno}")
+    case _ => Some(UnsupportedAnnotation(anno.getClass.getSimpleName, anno.toString))
```

Comment:
I understand that it is nice if we can just continue. However, the fact that we throw this `NotImplementedError` has gotten me a good number of issues and PRs which allowed me to check whether we can ignore the annotation or not.

Maybe a good compromise would be to generate the `UnsupportedAnnotation` as you do here, but then also add a `println` that tells users about the unsupported annotation and urges them to file an issue at https://github.com/ucb-bar/chiseltest/issues.